Update password in sync manager (#5898)
rotki/rotki: docs/changelog.rst

* :bug:`-` PnL report will now correctly show progress bar percentage if user has connected but non-syncing exchanges.
* :bug:`5749` Phishing zero token transfer transactions will now be hidden and ignored.
* :bug:`5717` Swaps will now be processed in accounting correctly even if set manually.

> * :bug:`-` When premium users change their rotki password, they will no longer have to reloging to be able to pull remote data.

Reviewer: ```suggestion
* :bug:`-` If a premium user changes their rotki password they will now be able to pull remote data without restarting the app
```
